src/pages/SeriesDetailsPage.js

```diff
@@ -0,0 +1,128 @@
+import { useParams, Link } from 'react-router-dom';
+import styled from 'styled-components';
+
+import useDetails from '../hooks/useDetails';
+import Poster from '../components/Poster';
+import ScreenReaderOnly from '../components/ScreenReaderOnly';
+import { ReactComponent as ArrowBackIcon } from '../assets/icons/arrow_back.svg';
+
+const {
+  REACT_APP_API_BASE_SERIES_URL,
+  REACT_APP_API_KEY,
+  REACT_APP_API_LANGUAGE,
+} = process.env;
+
+export default function SeriesDetailsPage() {
+  const { id } = useParams();
+  const seriesDetailsUrl = `${REACT_APP_API_BASE_SERIES_URL}/${id}?api_key=${REACT_APP_API_KEY}&language=${REACT_APP_API_LANGUAGE}`;
+  const { data: series } = useDetails(seriesDetailsUrl);
+
+  return (
+    <Wrapper>
+      <StyledLinkBack to="/serien">
+        <StyledArrowBackIcon />
+        <ScreenReaderOnly>Zurück</ScreenReaderOnly>
+      </StyledLinkBack>
+      <StyledBackdropImage backdropPath={series.backdrop_path} />
+      <StyledHeader>
+        <Poster
+          src={
+            series.poster_path
+              ? `https://image.tmdb.org/t/p/w300${series.poster_path}`
+              : require('../assets/images/poster.png')
+          }
+          alt={`${series.name}`}
+        />
+        <StyledHeaderText>
+          <h1>{series.name}</h1>
+          <h2>
+            {series.number_of_seasons} Staffeln -{' '}
+            {series.first_air_date
+              ? series.first_air_date.substr(0, 4)
+              : 'kein Datum vorhanden'}
+          </h2>
+        </StyledHeaderText>
+      </StyledHeader>
+      <StyledMain>
+        <h3>Handlung</h3>
+        <p>
+          {series.overview
+            ? series.overview
+            : 'Aktuell ist leider keine Beschreibung verfügbar'}
+        </p>
+      </StyledMain>
+    </Wrapper>
+  );
+}
+
+const Wrapper = styled.div`
+  position: relative;
+  max-width: 768px;
```

Comment:
gleiches Problem wie in den anderen files. probier mal aus ob du max-width überhaupt brauchst